filters/f_hwtransfer.c

```diff
@@ -291,8 +291,36 @@ static bool probe_formats(struct mp_filter *f, int hw_imgfmt, bool use_conversio
         if (!cur->av_device_ref)
             continue;
         cstr = av_hwdevice_get_hwframe_constraints(cur->av_device_ref, NULL);
-        if (!cstr)
-            continue;
+        if (!cstr) {
```

Comment:
also: I think it's worth noting this absence in the log
I'm just not sure if VERBOSE or WARN

Reply:
Added as VERBOSE.